homeassistant/components/openweathermap/weather_update_coordinator.py

```diff
@@ -179,10 +180,10 @@ def _convert_forecast(self, entry):
 
         return forecast
 
-    @staticmethod
-    def _fmt_dewpoint(dewpoint):
+    def _fmt_dewpoint(self, dewpoint):
         if dewpoint is not None:
-            return round(dewpoint / 100, 1)
+            dewpoint = dewpoint - 273.15
+            return self.hass.config.units.temperature(round(dewpoint, 1), TEMP_CELSIUS)
```

Comment:
I'd round after converting to Fahrenheit.
```suggestion
            return round(self.hass.config.units.temperature(dewpoint, TEMP_CELSIUS), 1)
```